File: JavaScript/6js 3/stringTransformer.js
/*
|--------------------------------------------------------------------------
| String transformer
|--------------------------------------------------------------------------
|
*/

function stringTransformer(str) {
   let x = str
      .split("")
      .map((char) => {
         return char === char.toLowerCase() ? char.toUpperCase() : char.toLowerCase();
      })
      .join("");
   return x.split(/(\s+)/).filter(Boolean).reverse().join("");
}


function stringTransformer(str) {
   return str
      .split(" ")
      .reverse()
      .join(" ")
      .split("")
      .map((v) => (v == v.toUpperCase() ? v.toLowerCase() : v.toUpperCase()))
      .join("");
}


function stringTransformer(str) {
   let s = "";
   for (let i = 0; i < str.length; ++i)
      if (str[i] == str[i].toLowerCase()) s += str[i].toUpperCase();
      else s += str[i].toLowerCase();
   return s.split(" ").reverse().join(" ");
}


console.log(stringTransformer("Example string"), "STRING eXAMPLE");

File: Python/6py 3/string_transformer.py
"""
|--------------------------------------------------------------------------
| String transformer
|--------------------------------------------------------------------------
|
"""

def string_transformer(s):
   x = ''.join([char.lower() if char.isupper() else char.upper() for char in s])
   return ' '.join(reversed(x.split(' ')))


def string_transformer(s):
   return ' ' .join(s.split(' ')[::-1]).swapcase()


def string_transformer(s):
   a = s.split(' ')
   res = []
   for i in a[::-1]:
      res.append(i.swapcase())
   return ' '.join(res)


print(string_transformer("Example string"), "STRING eXAMPLE")
print(string_transformer("Example Input"), "iNPUT eXAMPLE")
print(string_transformer("To be OR not to be That is the Question"), "qUESTION THE IS tHAT BE TO NOT or BE tO")
print(string_transformer(""), "")
print(string_transformer("You Know When  THAT  Hotline Bling"), "bLING hOTLINE  that  wHEN kNOW yOU")
print(string_transformer(" A b C d E f G "), " g F e D c B a ")